db/migrate/20220819183819_change_revenue_account_to_string_in_facility_accounts.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeRevenueAccountToStringInFacilityAccounts < ActiveRecord::Migration[6.1]
+  def up
+    change_column :facility_accounts, :revenue_account, :string
```

Comment:
Looks like this change will probably have the most implications in Northwestern.

Reply:
What happens to existing data when this is run?  I think this change should happen in the OSU repo, no need to change this for other schools if they dont need it

Reply:
I tested migrating this up and down on existing data. It's converted between strings and integers, and doesn't seem to have any issues.